telemetry/convert.py

```diff
@@ -36,6 +37,8 @@ class Converter:
     """A class for converting incoming payloads to a more compact form"""
     VERSION_UNCONVERTED = 1
     VERSION_CONVERTED = 2
+    # the unified FHR/Telemetry ping has a top-level version of 2 and payload version of 4
+    VERSION_UNIFIED = 2
```

Comment:
After converting the histograms to the compact form and creating a compatible "info" field, then you should set the post-conversion version to 2 (`VERSION_CONVERTED`).  
I'm not sure if we need another version to represent the post-conversion unified ping - it's starting to seem unweildy with so many different versions, and it seems like the point of this PR is to end up with payloads that are backwards-compatible with `VERSION_CONVERTED`.
Thoughts?